code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -51,5 +51,8 @@
 	name = "Ash Walker"
 	id = "ashlizard"
 	limbs_id = "unathi"
+	speedmod = -0.25

```

Comment:
Speed really doesn't work like this.

Reply:
Noted. I assumed it allowed that sort of partial. Makes sense. Back to the drawing board. Thanks!